### ISSUE TITLE
Rymek Citizen NPC in Bank spelling error

### ISSUE THREAD

**Issue by Qwaszx100** [Wynncraft/Issues#8941]
The Rymek Citizen NPC in the bank says bubby instead of buddy

![2017-12-22_01 41 29](https://user-images.githubusercontent.com/34765120/34290136-6e612afc-e6bb-11e7-94d5-6e8ad141bcf8.png)

Also, I went to go find the place the NPC spoke of and I think I found the right house

![2017-12-22_01 22 00](https://user-images.githubusercontent.com/34765120/34290193-b137a284-e6bb-11e7-8195-4988b24eb2f2.png)

And in the basement, there is a place that looks like it should be a teleport room but when I go inside of it nothing happens

Is this a bug or am I doing something wrong?

**Comment by ItzDennisz** [comment]
that was probably an exit.

**Comment by Qwaszx100** [comment]
I don't think it is an exit because if you look at this picture its right in the basement and there is no reason for there to be an exit there

![2017-12-22_08 53 43](https://user-images.githubusercontent.com/34765120/34302290-e40ab52c-e6f5-11e7-9232-cd6b32a2b3de.png)

**Comment by ItzDennisz** [comment]
yeah.. that's the rymek boss altar exit.

**Comment by Qwaszx100** [comment]
Then what was the NPC talking about?
